src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -590,7 +609,7 @@ private T IntroduceFrame<T>(BoundNode node, LambdaFrame frame, Func<ArrayBuilder
 
             var result = F(prologue, addedLocals);
 
-            _framePointers.Remove(frame);
+            //_framePointers.Remove(frame);
```

Comment:
I meant to just delete the line. Will fix. We need the frame pointers for when we do the local function reference pass later on.